### DISCUSSION TITLE
针对 CN，LGBTQIA 是否应该迎合国内的形态

### DISCUSSION THREAD

**Discussion by Nofated095** [LGBT-CN/LGBTQIA-In-China#197] [General]
如题，组织和项目都带有明确的 China 和 CN 字样。
那么得益于中国大陆奇妙的政治环境、社会制度和舆论氛围，LGBT 是否应该去迎合国内情况呢？
因为国内氛围实在是太差了，如果在保留 LGBT 的基础上，在一定方面去主动迎合国内的法律制度、社会舆论，也许 LGBT 会更好过？
这看上去是一个天真的想法，但我还是想讨论讨论

**Comment by ZhenQi1688** [comment]
并不太真。任何想法都是可能行动起起来的序言
